classes/class-admin.php

```diff
@@ -25,6 +25,11 @@ class Admin {
 	public $live_update;
 
 	/**
+	 * @var Export
+	 */
+	public $export;
```

Comment:
Since `Exports` relies on `List_Table` for filters, it doesn't really make sense. Let's keep it in `Admin`.